### ISSUE TITLE
<50% of the facts in the "fun facts" section on the about page are fun.

### ISSUE THREAD

**Issue by sjhsieh** [mldangelo/personal-site#168]
This is a very concerning problem. Is a fix for this planned in a future release?

**Comment by edwardkearney** [comment]
I one made a done for burning man. It costs much more than I hoped. It’s now with some random in Gerlach.

**Comment by edwardkearney** [comment]
2. I bought 5 80 dollar cashmere hats for 2 dollars each

**Comment by edwardkearney** [comment]
3. I am wearing J. Crew Hammerhead shark boxers, 484 stretch khakis, a broken in v-neck, a blue Oxford

**Comment by edwardkearney** [comment]
4. The best way to wake me up in the morning is to deflate my air mattress

**Comment by edwardkearney** [comment]
5. People in Moscow walk quickly because they are Russian.

**Comment by sjhsieh** [comment]
This still hasn't been fixed, rendering this project unusable until this issue has been resolved. I dug into the problem further:

"I have a list of thousands of ideas, like creating matching bow ties for cats and humans."
Starts out promising. Cute idea. 7/10

I almost always have a sketchbook with me.
I am skeptical. Always is a very strong word. 5/10

I can't locate every country on a map.
Applies to 9+/10 people. Low effort. 2/10

I added this page because so many people complained that my site was just a resume.
That's what this site will remain if these facts stay unfun 1/10
